spec/core/ics-002-client-semantics/README.md

```diff
@@ -429,26 +429,40 @@ type verifyNextSequenceRecv = (
   => boolean
 ```
 
-#### Optional Functions
+`verifyMembership` is a generic proof verification method which verifies a proof of the existence a value at a given `CommitmentPath` at the specified height.
```

Comment:
The names of these functions, i.e., `verifyMembership` and `verifyNonMembership`, are already used in ICS2 (see the Implementation strategies section). Also, both names are used in ICS23, but I guess that's okay.

Reply:
Clarified that the example in this section is the ICS-23 methods